code/modules/mob/living/basic/jungle/seedling/seedling.dm

```diff
@@ -0,0 +1,346 @@
+#define SEEDLING_STATE_NEUTRAL 0
+#define SEEDLING_STATE_WARMUP 1
+#define SEEDLING_STATE_ACTIVE 2
+
+//A plant rooted in the ground that forfeits its melee attack in favor of ranged barrages.
+//It will fire flurries of solar energy, and occasionally charge up a powerful blast that makes it vulnerable to attack.
```

Comment:
oh yeah this is probably all wrong now too

```suggestion
/**
 * A mobile plant with a rapid ranged attack.
 * It can pick up watering cans and look after plants.
 */
```